tests/HL7Test.php

```diff
@@ -8,6 +8,7 @@
 use Aranyasen\HL7;
 use Aranyasen\HL7\Segments\PID;
 use DMS\PHPUnitExtensions\ArraySubset\Assert;
+use Exception;
```

Comment:
The docblock for `empty_subfields_can_be_retained_if_needed()` says it throws `Exception`. This uses that class to avoid ambiguity.